packages/modules/web-viewer-react/src/services/Initializer.ts

```diff
@@ -36,14 +37,13 @@ export class WebInitializer {
   };
 
   /** Web viewer startup */
-  public static async startWebViewer(options: WebInitializerParams) {
+  public static async startWebViewer(options: WebInitializerParams, authClient?: ViewerAuthorizationClient) {
```

Comment:
could you just do a union for the type here to not add an extra arg?

Reply:
I think this should go into the `WebInitializerParams` type in types.ts?